Sources/TuistDependenciesTesting/Carthage/MockCarthageInteractor.swift

```diff
@@ -24,28 +21,33 @@ public final class MockCarthageInteractor: CarthageInteracting {
         )
 
         invokedFetch = true
-        invokedFetchCount += 1
-        invokedFetchParameters = parameters
-        invokedFetchParametersList.append(parameters)
-        if let error = stubbedFetchError {
-            throw error
-        }
+        try fetchStub?(parameters)
+    }
+
+    var invokedUpdate = false
+    var updateStub: ((UpdateParameters) throws -> Void)?
```

Comment:
I'd rather keep the lightweight solution of a "triple" rather than creating a helper struct for a stub